src/dstore/dstore_base.c

```diff
@@ -134,6 +164,8 @@ static int dstore_obj_shrink(struct dstore_obj *obj,  size_t old_size,
 	off_t offset;
 	char *tmp_buf = NULL;
 
+	perfc_trace_inii(PFT_DSTORE_OBJ_SHRINK, PEM_DSTORE_TO_NFS);
+
```

Comment:
Suggestion: do we need to trace old_size and new_size?

Reply:
Might be needed, can you suggest a use case where we would need these sizes?

Reply:
By observing those two params, we can compare the time consumed by such calls and relate the execution time with the size args.